### PR TITLE
allow small overshoot of power use for traveling 

### PR DESCRIPTION
**Pull request by Glease**
fix https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/14456

### CODE REVIEW

File: src/main/java/crazypants/enderio/teleport/TravelController.java [hunk 1]
```diff
@@ -111,42 +111,55 @@ private TravelController() {
...
+                if (!anchor.canBlockBeAccessed(toTp) && isValidTarget(toTp, target, TravelSource.BLOCK)) {
+                    return "not valid target";
+                }
+                return null;
             case STAFF:
             case STAFF_BLINK:
-                if (equippedItem == null || !(equippedItem.getItem() instanceof IItemOfTravel)) return false;
-                if (!((IItemOfTravel) equippedItem.getItem()).isActive(toTp, equippedItem)) return false;
+                if (equippedItem == null || !(equippedItem.getItem() instanceof IItemOfTravel)) return "not staff";
+                if (!((IItemOfTravel) equippedItem.getItem()).isActive(toTp, equippedItem)) return "staff not active";
```
Comment (BlueWeabo) [inline src/main/java/crazypants/enderio/teleport/TravelController.java:148]: Any reason for not using jabel here? the instanceof can make a variable of the class you want and wouldn't need the extra brackets to convert it to call the correct method.

File: src/main/java/crazypants/enderio/teleport/TravelController.java [hunk 2]
```diff
@@ -111,42 +111,55 @@ private TravelController() {
...
+                    return "not enough power";
+                }
+                return null;
             case TELEPORT_STAFF_BLINK:
             case TELEPORT_STAFF:
                 // tp staff is creative version of traveling staff
                 // no energy check or anything else needed
                 // but the player must actually be equipped with one of these
-                return equippedItem != null && equippedItem.getItem() instanceof ItemTeleportStaff;
+                if (equippedItem != null && equippedItem.getItem() instanceof ItemTeleportStaff) {
```
Comment (BlueWeabo) [inline src/main/java/crazypants/enderio/teleport/TravelController.java:159]: shouldn't the instanceof here be inverted if we are looking if the current equipped is a creative staff?

File: src/main/java/crazypants/enderio/teleport/TravelController.java (hunk 1 above)
Comment (Glease) [reply src/main/java/crazypants/enderio/teleport/TravelController.java:148]: this project does not yet have jabel enabled.

File: src/main/java/crazypants/enderio/teleport/TravelController.java (hunk 2 above)
Comment (Glease) [reply src/main/java/crazypants/enderio/teleport/TravelController.java:159]: fixed

File: src/main/java/crazypants/enderio/teleport/TravelController.java (hunk 1 above)
Comment (BlueWeabo) [reply src/main/java/crazypants/enderio/teleport/TravelController.java:148]: I see. Then it makes sense

**Review by BlueWeabo — APPROVED** [review]
Thanks for the fix

**Pull request merged**